modules/formulize/templates/admin/element_optionlist.html

```diff
@@ -35,11 +41,12 @@
 </fieldset>
 
 <script type="text/javascript">
+  
   $("[name=addoption]").click(function (){
     number = $(".useroptions:last").attr('name');
     number = parseInt(number) + 1;
     appendContents1 = '<{if $content.type == "radio"}><input type="radio" name="defaultoption" value='+number+'></input><{else}><input type="checkbox" name="defaultoption['+number+']" value='+number+'></input><{/if}>';
-    appendContents2 = '<input type="text" name="ele_value['+number+']"></input>';
+    appendContents2 = '<input type="text" name="ele_value['+number+']" onchange="checkForNAMES()"></input>';

```

Comment:
The actual function is checkForNames() without all-caps on NAMES, so this results in a javascript warning because the function can't be found. However, only the first text box needs to be changed anyway, so this javascript event can be removed anyway because it's not needed.